.env

```diff
@@ -58,10 +58,12 @@ SPDK_CENTOS_REPO_VER="9.0-21.el9"
 
 # Ceph Cluster
 CEPH_CLUSTER_VERSION="${CEPH_VERSION}"
+#CEPH_CLUSTER_VSTART_VERSION="v${CEPH_VERSION}"
+CEPH_CLUSTER_VSTART_VERSION="main"
 CEPH_BRANCH=ceph-nvmeof-mon
 CEPH_SHA=a4f2962ef7ad30b6e015c14e2c398015a766695a
-CEPH_VSTART_ARGS="--memstore"
```

Comment:
Any reason to remove `--memstore` flag for vstart cluster?

Reply:
This was already defined inside the [Dockerfile.ceph](https://github.com/ceph/ceph-nvmeof/blob/devel/Dockerfile.ceph#L101).